src/doc_scan_service/session/create/session.specification.builder.js

```diff
@@ -154,6 +154,12 @@ class SessionSpecificationBuilder {
     return this;
   }
 
+  withIdentityProfile(identityProfile) {
```

Comment:
Sorry for being pedantic and annoying, I checked also on the .NET version, and this should be called `withIdentityProfileRequirements`, which then means that `identityProfile` should become `identityProfileRequirements`...